corehq/apps/cloudcare/static/cloudcare/js/markdown.js

```diff
@@ -140,7 +140,18 @@ hqDefine('cloudcare/js/markdown', [
             // lazy init to avoid dependency order issues
             md = initMd();
         }
-        return md.render(DOMPurify.sanitize(text || "").replaceAll("&#10;", "\n"));
+        var rendered = md.render(DOMPurify.sanitize(text || "").replaceAll("&#10;", "\n"))
```

Comment:
I think it's preferred to use `let` if a variable is going to be mutated and `const` otherwise.